### PR TITLE
Funny things

### PR DESCRIPTION
**Pull request by Grummfy**

### CODE REVIEW

**Comment by Grummfy** [comment]
et voilà.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 /build
 /tests/vendor
+.idea
```
Comment (marmotz) [inline .gitignore]: Please add this IDE dependant lines in your global gitignore configuration.
If everybody add it's own configuration, this should be very ugly...

Comment (Grummfy) [inline .gitignore]: corrigé. désolé , oubli de ma part

**Comment by marmotz** [comment]
Thanks

**Pull request merged**